src/lib/components/Layers/Layers.js

```diff
@@ -120,15 +144,12 @@ export class Layers extends Component {
       }
       return null;
     });
-
-    return <ul className="layers">{layerItem.length > 0 ? layerItem : ifPermissionDenied()}</ul>;
+    return <ul className="layers">{layerItem}</ul>;
   }
 }
-
 Layers.propTypes = {
   mapId: PropTypes.string.isRequired,
   layers: PropTypes.arrayOf(PropTypes.any).isRequired,
   currentRegion: PropTypes.string,
 };
-
-export default Layers;
+export default connect(mapStateToProps)(Layers);
```

Comment:
@cKellyDesign I tested this with spotlight-zimbabwe and idai-recovery and I'd say it is backward compatible

Reply:
@kahummer that's a good sign! Can we also test on a few other older repos? Like maybe service mapping?

Reply:
@kahummer I think @ckellyDesign is right on this because older versions of `gisida` don't have the `toggleGroups ` action and right now it is needed to toggle layers. Are you sure you tested using older versions of `gisida`?

Reply:
@ciremusyoka I think most sites won't work when using older versions of gisida and new versions of gisida-react and vice-versa reason being older versions of gisida had different structure of the store and a lot of changes have been made that make the versions different from what we currently have. I think compatibility of older versions of gisida to newer versions of gisida-react is a tall order and what seems achievable is having same versions being compatible e.g version 1.2.0 gisida to 1.2.0 gisida-react  instead of 1.2.0 gisida to 1.3.5 gisida-react. The aim here should be having this branch compatible with staging and to ensure it supports all clients. @cKellyDesign please share your thoughts on this

Reply:
> ...a lot of changes have been made that make the versions different from what we currently have.

I think it's safe to say that such changes did not always (or often) have backwards compatibility in mind.

As a team that continuously integrates new development into multiple active repositories, version compatibility should **at least** be updated in`gisida-react`'s (peer?)Dependencies (eg - `"gisida": "^1.3.x"`) **and** noted in release notes (eg - `gisida-react@1.3.y` requires at least `gisida@1.3.x`)

___

@kahummer @ciremusyoka @kelvin-muchiri - Also, to be clear, `backwards compatibility` should be considered in previous versions of a single repo. Consider:
* client has `gisida-react@1.3.x` installed
(in this case `x` is any version with a non-connected `Login` component)
* client runs `yarn add gisida-react@1.4.0 && yarn start`
(but client does not update `src/index.js` or anything else)
* client site breaks b/c the (connected) `Login` component in `1.4.0` is getting the wrong props (b/c it's getting props meant for the old non-connected component)

In this scenario (which is relevant to this PR), the `Login` component in `1.4.0` should actually work with props meant for `1.3.x`, this can probably be achieved by adding more logic to `mapStateToProps`. Only in `2.0.0` would we drop backwards compatibility for `1.x.x` and officially deprecate support for the old non-connected `Login` props as a part of a major release. This is the way.

If this type of backwards compatibility is not feasible at this point, then let's just be sure to include notes on the incompatibility (and how to upgrade). Such a note can live in this PR for now and be added to release notes when the next release is published.

cc @KipSigei @moshthepitt @pld

Reply:
How onerous is the compatibility fix suggested by @cKellyDesign? We want to avoid incompatibility within major versions, and must note any we introduce within or between any versions.

Reply:
> backwards compatibility should be considered in previous versions of a single repo.
> 

This isn't hard to achieve. Still testing various client projects.